### PR TITLE
CROSSTOOL docs: fix formatting errors

### PR DESCRIPTION
**Pull request by laszlocsomor**

### CODE REVIEW

File: site/docs/crosstool-reference.md [hunk 1]
```diff
@@ -156,7 +156,10 @@ support and expansion. These are:
    </td>
   </tr>
   <tr>
-   <td><code>requires: ['feature1', 'feature2']</code>
+   <td><pre>requires: {
```
Comment (hlopko) [inline site/docs/crosstool-reference.md]: Drop colon please.

File: site/docs/crosstool-reference.md [hunk 2]
```diff
@@ -168,30 +171,31 @@ support and expansion. These are:
...
-       This improves error handling by listing the alternatives if the user asks
-       for two or more mutually exclusive features at once.
+       specify <code>provides: "sanitizer"</code>.</p>
+       <p>This improves error handling by listing the alternatives if the user asks
+       for two or more mutually exclusive features at once.</p>
    </td>
   </tr>
   <tr>
-   <td><code>with_feature: {feature: 'feature1', not_feature: 'feature2']</code>
+   <td><pre>with_feature: {
```
Comment (hlopko) [inline site/docs/crosstool-reference.md]: Also here (drop colon) pls.

**Comment by hlopko** [comment]
Looks fantastic, thank you so much! Only nits for consistency. LGTM!

**Comment by laszlocsomor** [comment]
Thank you for the swift review!

I see no comments though, did you intend to send none?

**Comment by hlopko** [comment]
Hups :)

File: site/docs/crosstool-reference.md (hunk 2 above)
Comment (laszlocsomor) [reply site/docs/crosstool-reference.md]: Done.

File: site/docs/crosstool-reference.md (hunk 1 above)
Comment (laszlocsomor) [reply site/docs/crosstool-reference.md]: Done.

**Comment by laszlocsomor** [comment]
Done. I also removed the line-ending "," in the `requires { ... }` and `with_feature { ... }` messages.

**Pull request closed without merging**